### DISCUSSION TITLE
Measure of variability around PK analysis?

### DISCUSSION THREAD

**Discussion by eburg007** [Open-Systems-Pharmacology/Forum#852] [Questions & Problems] [answered]
I'm wondering if it's possible to find--or create a new feature, if it doesn't already exist--a measure of variability (standard error, confidence interval, etc.) surrounding the PK parameter calculations within a simulation (AUC, Cmax, etc.)?

For example: when I run a parameter identification I can see the 5-95% CI for that varied parameter, but once I apply the optimized result to my simulation, there's no way to calculate uncertainty in the "new" PK analysis. It would be nice to be able to see the 5-95% estimates for AUC, Cmax, etc., especially when you're trying to compare a model result to published clinical data.

Thanks!

**Comment by msevestre** [comment]
@eburg007 moving this to the forum for now until we decide whether it's a feature request or not :) 
Thanks

**Comment by StephanSchaller** [comment, marked as answer]
Of course, as always, this could be done through R: export the correlation/covariance matrix and sample from it for a population simulation

In PK-Sim sth "similar" could be achieved by just creating the confidence interval plots (one of the results plots for a parameter identification)?... this should directly display the simulations with CI against data.

To do sth. as described above in PK-Sim, we would need to create a population which captures covariance on-the-fly when applying the results to the simulations which would then have to be plugged in to the simulations... not impossible, but also not planned at this time...
